Sources/SecureXPC/Server/XPCMachServer.swift

```diff
@@ -197,6 +200,18 @@ internal class XPCMachServer: XPCServer, NonBlockingStartable {
 		return accept
 	}
 
+    public override var endpoint: XPCServerEndpoint {
+        guard let connection = self.serviceListenerConnection else {
+            fatalError("An XPCServer's endpoint can only be retrieved after start() has been called on it.")
+        }
+
+        let endpoint = xpc_endpoint_create(connection)
```

Comment:
I'm assuming this actually works? I don't have any good way to test it, but just realized this is rather unintuitive because the connection being serialized into an endpoint here is the one that accepts new connections, not one which can be used to receive events.

Reply:
IIRC, it's "smart" and doesn't encode the service listener connection itself. When a connection it created from this, it's able to talk to the mach service.

However, this isn't something I can test right now, either

Reply:
Ah, what's stopping you from trying this out? I assume it doesn't actually need to be sent across processes in order to be tested (although normally not doing so would be pointless).

Reply:
How do you think we should proceed? I'm moving to a new house soon, so I won't be able to build out a test bed for this any time soon. I see two options:

1) Yolo ship it, follow up on it later (with an issue to track it)
2) Fatal-error this code path to say that it's untested, and treat it as effectively not-yet-implemented.

Reply:
Let's go with option 2 for now. I'd volunteer to try testing it out, but between end of year busyness at work and heading off on holiday soon I just won't have time before early January. Can revisit it then.

Reply:
Opened https://github.com/trilemma-dev/SecureXPC/issues/33